### ISSUE TITLE
How did you generate the train and test  txt files?

### ISSUE THREAD

**Issue by alanlisten** [xiangwang1223/knowledge_graph_attention_network#11]
Hi, I noticed that the user id of the training set and the test set is exactly the same, except for the item id. I think this is a more complicated preprocessing process, but I did not find the corresponding code in the code file.

Thank you :)

**Comment by xiangwang1223** [comment]
Thanks for your interest. We filter the original dataset by ourselves and do not release the data laundury code.
